pkg/handlers/handlers.go

```diff
@@ -35,7 +35,7 @@ func NewHandler() *Handler {
 func init() {
 	kotsscheme.AddToScheme(scheme.Scheme)
 	troubleshootscheme.AddToScheme(scheme.Scheme)
-	veleroscheme.AddToScheme(scheme.Scheme)
+	velerov1.AddToScheme(scheme.Scheme)
```

Comment:
Since all the velero object interactions now rely on:
- https://github.com/replicatedhq/kots/blob/a422a554bd22d7b1e7e0feed4f76e33d77f0fc57/pkg/k8sutil/clientset.go#L157-L171

Which has the scheme set via:
- https://github.com/replicatedhq/kots/blob/a422a554bd22d7b1e7e0feed4f76e33d77f0fc57/pkg/k8sutil/clientset.go#L36-L40

We probably don't need this anymore. Nevertheless keeping it for now just be cautious. Once we refactor our client usage across the board and drop a lof of our clientset usages we can take care of removing these.